### PR TITLE
Add choropleth-us

### PR DESCRIPTION
**Pull request by bobhaslett**

### CODE REVIEW

File: choropleth-us/choropleth.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import * as d3 from 'd3';
+import * as gChartcolour from 'g-chartcolour';
+import topojson from 'topojson-client';
+
+export function draw() {
+    // let interpolation =d3.curveLinear
+    let colourScale = d3.scaleThreshold();
+    let level = 'counties';
+    let rem = 16;
+    var projection = d3.geoIdentity()
```
Comment (aendrew) [inline choropleth-us/choropleth.js]: Change to `const`

File: choropleth-us/choropleth.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                .scale(0.67)
+
+    function chart(parent) {
+
```
Comment (aendrew) [inline choropleth-us/choropleth.js]: Extraneous whitespace

File: choropleth-us/choropleth.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        if (level === 'counties') {
+            counties.selectAll('path')
+            .data(d => topojson.feature(d, d.objects.counties).features)
+            .enter()
+            .append('path')
+            .attr('fill', function(d) {
+                if(d.properties[0].value === 'none') {
+                    return 'none'
+                }
+                else { return colourScale(d.properties[0].value)}
```
Comment (aendrew) [inline choropleth-us/choropleth.js]: Use arrow functions if `this` keyword isn't used in function.

File: choropleth-us/choropleth.js (hunk 1 above)
Comment (bobhaslett) [reply choropleth-us/choropleth.js]: Doesn't that mean they can't be changed by the getter/setters?

Comment (aendrew) [reply choropleth-us/choropleth.js]: @bobhaslett Ah, right, apologies — I mean `let`.

**Pull request merged**